src/test/java/org/openmrs/contrib/qaframework/page/ActiveVisitsPage.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * <p>
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
```

Comment:
@sherrif10 I have observed this trade made in many of the OpenMRS repos and I  want to think this is the right way of writing trade mark. 

Would be great to have all the files in this PR having this trade mark and am wondering whether are you able to add this trade mark all through the files apart from pom.xml ?